frontend/.env.development

```diff
@@ -7,3 +7,4 @@ REACT_APP_STREAM=$DOMAIN/stream
 REACT_APP_TOKEN=$DOMAIN/auth/token
 REACT_APP_LOG=$DOMAIN/log
 REACT_APP_SIGNUP=$DOMAIN/user/sign-up
+REACT_APP_STREAMLOGS=$DOMAIN/streamlogs
```

Comment:
The correct endpoint is `/log/stream`

You are appending the stream ID correctly in LogApi tho!

Reply:
Thanks for letting me know! I think the old endpoint was `/streamlogs` and I didn't know it got changed.